lib/data_magic/document_builder.rb

```diff
@@ -47,7 +47,12 @@ def map_field_types(row, valid_types, field_types = {}, null_value = 'NULL')
           if value == null_value
             mapped[key] = nil
           else
-            type = field_types[key.to_sym] || field_types[key.to_s]
+            tmp = {"age" => "integer", "height" => "float"}
```

Comment:
What's this addition about? Looks like it's related to the test you added to `api_spec.rb` - is that right?